src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -80,130 +80,130 @@
 
     <sys:String x:Key="Deploy_Deployment">Distribuzione di Office</sys:String>
-    <sys:String x:Key="Deploy_Deployment.Description">Prima di distribuire Office, assicurarsi che la modalità di distribuzione sia impostata correttamente e che il prodotto e la lingua siano configurati.</sys:String>
-    <sys:String x:Key="Deploy_Start_Deploy">Avvia _Distribuzione</sys:String>
-    <sys:String x:Key="Deploy_Reset">_Reimposta tutte le impostazioni</sys:String>
+    <sys:String x:Key="Deploy_Deployment.Description">Prima di distribuire Office, assicurati che la modalità di distribuzione sia impostata correttamente e che siano configurati il prodotto e la lingua.</sys:String>
+    <sys:String x:Key="Deploy_Start_Deploy">Avvia _distribuzione</sys:String>
+    <sys:String x:Key="Deploy_Reset">_Ripristina tutte le impostazioni</sys:String>
     <sys:String x:Key="Deploy_View_Code">_Visualizza codice XML</sys:String>
     <sys:String x:Key="Deploy_Apply_Preferences">_Applica le preferenze per le applicazioni di Office</sys:String>
     <sys:String x:Key="Deploy_Export_Config">_Esporta configurazione</sys:String>
     <sys:String x:Key="Deploy_Import_Config">_Importa configurazione</sys:String>
-    <sys:String x:Key="Deploy_Import_Config_Web">Importa configuration dal _Web</sys:String>
-    <sys:String x:Key="Deploy_Mode">Modalità di distribuzione</sys:String>
-    <sys:String x:Key="Deploy_Mode.Edit">Modifica la configurazione</sys:String>
-    <sys:String x:Key="Deploy_Mode.Download">Scarica</sys:String>
+    <sys:String x:Key="Deploy_Import_Config_Web">Importa configurazione dal _web</sys:String>
+    <sys:String x:Key="Deploy_Mode">Modalità distribuzione</sys:String>
+    <sys:String x:Key="Deploy_Mode.Edit">Modifica configurazione</sys:String>
+    <sys:String x:Key="Deploy_Mode.Download">Download</sys:String>
     <sys:String x:Key="Deploy_Mode.Install">Installa</sys:String>
-    <sys:String x:Key="Deploy_Mode.Create_ISO">Crea un file ISO</sys:String>
+    <sys:String x:Key="Deploy_Mode.Create_ISO">Crea file ISO</sys:String>
     <sys:String x:Key="Deploy_Architecture">Architettura</sys:String>
-    <sys:String x:Key="Deploy_Architecture_Current_Installed">Architettura di Office attualmente installata: {0}.</sys:String>
+    <sys:String x:Key="Deploy_Architecture_Current_Installed">Architettura Office attualmente installata: {0}.</sys:String>
     <sys:String x:Key="Deploy_Architecture.x86">32 bit</sys:String>
     <sys:String x:Key="Deploy_Architecture.x64">64 bit</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32-bit compatibile)</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64-bit compatibile)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32bit compatibile)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64bit compatibile)</sys:String>
```

Comment:
You changed `32-bit` to `32bit`, the line 134 and 135 should be changing too? I'm not sure, please check.

In the source English, the line 134 and 135 is `32-bit` and `64-bit`.

Reply:
Fixed (remove space in line 134 e 135)